docker-compose.yml

```diff
@@ -43,5 +43,31 @@ services:
   redis:
     build: ./redis
 
+  localstack-container:
+    container_name: "localstack-container"
+    privileged: true
+    image: localstack/localstack:0.11.0
+    ports:
+      - "4566-4599:4566-4599"
+      - "8081:8081"
```

Comment:
Do we need all these ports published to the host? Based on the CKAN config it looks like 4572 is the only one that needs to be accessible by the host.

Reply:
Did you want to add a code comment about this?